iota/api.py

```diff
@@ -940,6 +954,11 @@ def send_trytes(self, trytes, depth, min_weight_magnitude=None):
 
       If not provided, a default value will be used.
 
+    :param security_level:
+      Number of iterations to use when generating new addresses (see get_new_addresses).
+      This value must be between 1 and 3, inclusive.
+      If not set, defaults to AddressGenerator.DEFAULT_SECURITY_LEVEL = 2
```

Comment:
❔ I think this was meant to be added to the docstring for `send_transfer`, not `send_trytes`.

Reply:
oops...